src/test/java/space/obminyashka/items_exchange/controller/UserControllerTest.java

```diff
@@ -79,94 +79,34 @@ void negativeTestReceivingInformationAboutAnotherUser() throws Exception {
 
     @Test
     @WithMockUser(username = "admin")
-    void updateUserInfo_shouldReturn403WhenUsernameIsChanged() throws Exception {
-        when(userService.findByUsernameOrEmail(any())).thenReturn(Optional.of(user));
-        when(userService.update(any(), any())).thenThrow(new IllegalOperationException(
-                getMessageSource("exception.illegal.field.change") + user.getUsername()));
-
-        MvcResult result = getResultActions(HttpMethod.PUT, "/user/info",
-                createUserDtoForUpdatingWithChangedUsernameWithoutPhones(), status().isForbidden())
-                .andDo(print())
+    void updateUserInfo_InvalidFirstName_shouldReturn400WhenShortFirstName() throws Exception {
```

Comment:
That's logic it's not enough for test purposes. You get expected code response but don't know for sure which response message was trown. That's why you have to validate the message as well (please re-check all updated tests).